tests/Resources/ti.ui.shortcutitem.test.js

```diff
@@ -11,13 +11,13 @@
 
 const should = require('./utilities/assertions');
 
-describe.android('Titanium.UI.ShortcutItem', () => {
-
+describe('Titanium.UI.ShortcutItem', () => {
+  
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.shortcutitem.test.js;
  Line: 15;
-->
- :no_entry_sign: tests/Resources/ti.ui.shortcutitem.test.js line 15 – Trailing spaces not allowed. (no-trailing-spaces)